src/Pages/Login/Login.scss

```
@@ -0,0 +1,105 @@
+@import '../../Styles/common.scss';
+
+.login {
+  @extend %flex-all-center;
+  width: 100vw;
+  height: 100vh;
+  margin: 50px 0px;
+  padding: 0px 0px 50px;
+
+  .loginContainer {
+    padding: 50px 0px;
+    text-align: center;
+
+    h2 {
+      color: #222222;
+      margin-bottom: 30px;
+      font-size: 30px;
+    }
+    form {
+      display: flex;
+      flex-direction: column;
+
+      img:nth-child(2),
+      img:nth-child(5) {
+        position: relative;
+        width: 20px;
+        height: 20px;
+        bottom: 35px;
+        right: 130px;
+      }
+      .far {
+        position: relative;
+        left: 130px;
+        bottom: 33px;
+        font-size: 15px;
+      }
+      .inputId {
+        width: 100%;
+        height: 50px;
+        padding: 0px 0px 0px 44px;
+        border: 1px solid #bebebe;
+      }
+      .inputId:focus {
+        border: 1px solid black;
+      }
+      .inputId.inputFail {
+        border: 1px solid red;
+      }
+      .inputId.inputSuccess {
+        border: 1px solid green;
+      }
+      .far {
+        border: none;
+      }
+      .far.inputFail {
+        color: red;
+      }
+      .far.inputSuccess {
+        color: green;
+      }
+      .loginSave {
```

```suggestion
      }
      
      .far {
        border: none;
      
        &.inputFail {
          color: red;
        }
      
        &.inputSuccess {
          color: green;
        }
      }
      .loginSave {
```
- 새로운 셀렉터 앞에서는 한줄 띄워주세요
- `&`를 사용하여 공통 부분에 대해서 nesting 해주세요

---

&, 정말 유용합니다, 감사합니다...!